### ISSUE TITLE
Display list items in one line each when rendering inside cards

### ISSUE THREAD

**Issue by danicatalan** [huridocs/uwazi#1265]
Display each item in one line (as a bulleted list, not an inline one) when rendering properties with multiple values on cards:

![image](https://user-images.githubusercontent.com/6364153/29708743-14b47ae2-898a-11e7-9355-c4f8a075b0c1.png)

## Extra ball

Should we do the same on sidepanel lists?

![image](https://user-images.githubusercontent.com/6364153/29708759-26606260-898a-11e7-9889-94b209d44372.png)

**Comment by RafaPolit** [comment]
Why are we discussing this again? What has changed?

In the past we have discussed this and concluded that long lists would render horribly.  For instance, the Descriptors found in CEJIL Processes, they could make for an incredibly long list.

@txau @danicatalan and me were on that discussion when doing the dates lists and we actually implemented exceptions to prevent lists from going the bullet way on cards.
